content/backend/graphql-java/4-connectors.md

```diff
@@ -137,7 +136,6 @@ For this project, you'll use MongoDB as the persistent storage, but by following
 	                doc.get("_id").toString(),
 	                doc.getString("url"),
 	                doc.getString("description"),
-	                doc.getString("postedBy"));
```

Comment:
Simply removing the line would make it invalid syntax. Should end up like: `doc.getString("description"));` instead.